checker/requestcache.go

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) 2018 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package checker
+
+import (
+	"fmt"
+	"regexp"
+
+	authz "github.com/envoyproxy/data-plane-api/api/auth"
+	"github.com/projectcalico/app-policy/policystore"
+	"github.com/projectcalico/app-policy/proto"
+)
+
+// requestCache contains the CheckRequest and cached copies of computed information about the request
+type requestCache struct {
+	Request *authz.CheckRequest
+	store   *policystore.PolicyStore
+	source  *Peer
+}
+
+type Peer struct {
```

Comment:
nit: i don't think `Peer` is used outside of the `checker` package? if not then it could be all lowercase